sources/quattroshapes-gp.json

```diff
@@ -0,0 +1,12 @@
+{
+	"id": 1108970625,
+	"fullname": "Quattroshapes Point Gazetteer",
+	"name": "quattroshapes_gp",
```

Comment:
Taken care of in: https://github.com/whosonfirst/whosonfirst-sources/commit/cee74df632e7e7428fbdeee4fc3f28763720c48d.